client/lib/domains/index.js

```diff
@@ -116,6 +116,11 @@ function canRedirect( siteId, domainName, onComplete ) {
 		domainName = 'http://' + domainName;
 	}
 
+	if ( domainName.includes(`@`) ) {
```

Comment:
I think you have to use `lodash` includes because this one doesn't work in IE. Looking at the repo, [it seems that's the way to go](https://github.com/Automattic/wp-calypso/search?p=3&q=includes&type=).

Reply:
Good to know, @alshakero! I have updated to lodash's includes in 2ad388e962df13117e0a975edd77ed9da7bf722a.

Reply:
FWIW, we include shims that make it so both versions work. Personally prefer the more explicit lodash ones, but on the other hand that pulls in lodash to the JS fragment/section _and_ possibly the shim (since that's more global), so that's more code to (down)load for the user.

But we're already pulling in `includes` there from lodash, so it should be fine in this case.